### ISSUE TITLE
Story is not shown in storybook

### ISSUE THREAD

**Issue by Krausso** [storybookjs/storybook#14464]
![image](https://user-images.githubusercontent.com/52201020/113474326-0c671280-9478-11eb-88d0-2bbe4a011a2d.png)
Everything that I've written is on the screenshot above.

I'm not really sure what's wrong with it, cuz there's no errors
There was just an error that looks like 
```
Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined.
You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `unboundStoryFn`.
```
And the warning that my `export default` is actually unused.

**Comment by Krausso** [comment]
Already fixed 👌🏿

**Comment by nawal-neeraj** [comment]
How you fixed?

**Comment by Krausso** [comment]
> How you fixed?

Don't know, actually

**Comment by jefferson1104** [comment]
What is solution?

**Comment by krau5** [comment]
In my case, it was a problem with component's props

**Comment by guilhermefront** [comment]
For me I was using a component with a wrong import as a decorator, on the `preview.js` file.

```jsx
// ThemeProvider has the wrong import name
import { ThemeProvider } from '../src/styles/theme';

export const decorators = [
  (Story) => (
    <ThemeProvider>
      <Story />
    </ThemeProvider>
  ),
];
```

**Comment by aldrichdev** [comment]
> ```js
> // ThemeProvider has the wrong import name
> import { ThemeProvider } from '../src/styles/theme';
> ```

How is this wrong?

I am experiencing the same issue. Getting `Check the render method of unboundStoryFn` for every story

**Comment by blwinters** [comment]
This was a misleading error message in my case, but I guess it did point me to look at import issues. 

I'm developing for React Native and using Storybook via the React Native Web addon. I have a component that relies on a package that is unsupported on the web, `react-native-linear-gradient`, though I don't actually include that component in any stories. However, I had added that component to the directory index, so any story that imported a component via that index was also implicitly requiring the native-only component.

To fix this I added a dummy file called `TopShadow.web.tsx` (next to `TopShadow.tsx`), which returns a web-compatible view instead of requiring that gradient package. With that in place, React Native Web sources that file instead of the mobile implementation when building for the web/Storybook.
